src/MaterialModels.jl

```diff
@@ -64,7 +70,7 @@ When multithreading is used, each threads needs its own cache.
 Returns `nothing` for materials that don't need a cache.
 """
 function get_cache(::AbstractMaterial)
-    return nothing
+    nothing 
```

Comment:
I usually prefer being explicit with the return statements (and I think we pretty much are in the rest of the package).